src/components/forms/PasswordResetForm/PasswordResetForm.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Button, Form } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import usersActions from '../../../_actions/users.actions';
+
+class PasswordResetForm extends React.Component {
+  state = {
+    data: {
+      password: ''
+    }
+  };
+
+  handleDataChange = e => {
+    const { id, value } = e.target;
+    this.setState(prevState => ({
+      data: {
+        ...prevState.data,
+        [id]: value
```

Comment:
Could this be anything other than `password`? If so, please document it in state = { ... }.